demisto_sdk/main.py

```diff
@@ -272,6 +273,23 @@ def format(file_type, **kwargs):
     return 1
 
 
+@main.command(name="init", short_help="Initiate a new Pack, Integration or Script.")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-n", "--name", help="The name of the object you want to create")
+@click.option(
+    "-o", "--outdir", help="The output dir to write the object into")
+@click.option(
+    '--integration', is_flag=True, help="Create an Integration")
+@click.option(
+    '--script', is_flag=True, help="Create a script")
```

Comment:
mention that this will be the HelloWorld example

Reply:
done